tg_bot/modules/sql/rss_sql.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy import Column, UnicodeText, Integer
+
+from tg_bot.modules.sql import BASE, SESSION
+
+
+class RSS(BASE):
+    __tablename__ = "rss_feed"
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, nullable=False)
+    chat_id = Column(UnicodeText, nullable=False)
+    feed_link = Column(UnicodeText)
+    old_entry_link = Column(UnicodeText)
+
+    def __init__(self, user_id, chat_id, feed_link, old_entry_link):
+        self.user_id = user_id
+        self.chat_id = chat_id
+        self.feed_link = feed_link
+        self.old_entry_link = old_entry_link
+
+    def __repr__(self):
+        return "<RSS for {} with chatID {} at feed_link {} with old entry {}>".format(self.user_id,
+                                                                                      self.chat_id,
+                                                                                      self.feed_link,
+                                                                                      self.old_entry_link)
+
+
+def check_url_availability(tg_user_id, tg_feed_link, tg_chat_id):
+    row = SESSION.query(RSS).filter(RSS.user_id == tg_user_id,
```

Comment:
add try:/finally blocks to anything returning sql objects to ensure the session is eventually closed.
See other sql modules for more info